packages/insomnia-app/app/ui/components/codemirror/code-editor.tsx

```diff
@@ -92,13 +91,17 @@ const BASE_CODEMIRROR_OPTIONS: CodeMirror.EditorConfiguration = {
 
 export type CodeEditorOnChange = (value: string) => void;
 
-type ReduxProps = ReturnType<typeof mapStateToProps>;
-
-interface OwnProps {
+interface FCProps {
+  enableNunjucks?: boolean;
   ignoreEditorFontSettings?: boolean;
 }
 
-const mapStateToProps = (state: RootState, { ignoreEditorFontSettings }: OwnProps) => {
+const useDerivedProps = ({ enableNunjucks, ignoreEditorFontSettings }: FCProps) => {
+  const {
+    handleRender,
+    handleGetRenderContext,
+  } = useGatedNunjucks({ disabled: !Boolean(enableNunjucks) });
```

Comment:
```suggestion
  } = useGatedNunjucks({ disabled: !enableNunjucks });
```

I might be mistaken, but I believe we don't need the Boolean cast here

Reply:
Nice catch bc423eec98900cdda8112a2ffddd7096a4fbccca